src/test/static/namespaces/memberof-functions.js

```diff
@@ -13,7 +13,7 @@ Polymer.aaa = aaa;
  * @memberof Polymer
  */
 Polymer.bbb = function bbb() {
-
+  return;
```

Comment:
Why add this?

Reply:
Was to get more test coverage from this suite. But then I added my own suite, so I don't need it here too. Removed.